content/workflows/csp/aws/service/cloudtrail/resources/trail/policy-context/cloud-watch-logs-not-enabled.yaml

```
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: cloud-watch-logs-not-enabled
```

```suggestion
id: cloudwatch-logs-not-enabled
```

Please also rename file to not have a hypen between cloud and watch